README.rst

```diff
@@ -13,1837 +13,30 @@ Turn complex GraphQL queries into optimized database queries.
-produce a valid result. Therefore, the complete result list, shown here
-in JSON format, would be:
-
-.. code:: python
-
-    [
-        {"s_name": "a", "t_name": "x"},
-        {"s_name": "a", "t_name": "y"},
-        {"s_name": "b", "t_name": "x"},
-        {"s_name": "b", "t_name": "y"},
-    ]
-
-This is precisely what the :code:`MATCH` compilation target is guaranteed to
-produce. The remainder of this section is only applicable to the
-:code:`gremlin` compilation target. If using :code:`MATCH`, all of the queries
-listed in the remainder of this section will produce the same, complete
-result list.
-
-Since the :code:`gremlin` compilation target does not guarantee a complete
-result list, querying the database using a query string generated by the
-:code:`gremlin` compilation target will produce only a partial result list
-resembling the following:
-
-.. code:: python
-
-    [
-        {"s_name": "a", "t_name": "x"},
-        {"s_name": "b", "t_name": "x"},
-    ]
-
-Due to limitations in the underlying query language, :code:`gremlin` will by
-default produce at most one result for each of the starting locations in
-the query. The above GraphQL query started at the type :code:`S`, so each
-:code:`s_name` in the returned result list is therefore distinct.
-Furthermore, there is no guarantee (and no way to know ahead of time)
-whether :code:`x` or :code:`y` will be returned as the :code:`t_name` value in each
-result, as they are both valid results.
-
-Users may apply the :code:`@output_source` directive on the last scope of
-the query to alter this behavior:
-
-.. code::
-
-    {
-        S {
-            name @output(out_name: "s_name")
-            out_E @output_source {
-                name @output(out_name: "t_name")
-            }
-        }
-    }
-
-Rather than producing at most one result for each :code:`S`, the query will
-now produce at most one result for each distinct value that can be found
-at :code:`out_E`, where the directive is applied:
-
-.. code:: python
-
-    [
-        {"s_name": "a", "t_name": "x"},
-        {"s_name": "a", "t_name": "y"},
-    ]
-
-Conceptually, applying the :code:`@output_source` directive makes it as if
-the query were written in the opposite order:
-
-.. code::
-
-    {
-        T {
-            name @output(out_name: "t_name")
-            in_E {
-                name @output(out_name: "s_name")
-            }
-        }
-    }
-
-SQL
----
-
-Relational databases are supported by compiling to SQLAlchemy core as an intermediate
-language, and then relying on SQLAlchemy's compilation of the dialect-specific SQL query. The
-compiler does not return a string for SQL compilation, but instead a SQLAlchemy :code:`Query`
-object that can be executed through a SQLAlchemy `engine
-<https://docs.sqlalchemy.org/en/latest/core/engines.html>`__.
-
-Our SQL backend supports basic traversals, filters, tags and outputs, but there are still some
-pieces in development:
+Example
```

Comment:
Can we include an OrientDB example as well? This would be ideal since OrientDB is going to be easier to use (it's more mature) and more full-featured.

Reply:
Ok I'll add one.

Reply:
I added it on top of the SQL example since it shows query traversals. I'll move the SQL example to the top once we improve it a bit.